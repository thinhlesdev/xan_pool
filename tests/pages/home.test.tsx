import React from 'react';
import axios from 'axios';
import '@testing-library/jest-dom';
import MockAdapter from 'axios-mock-adapter';
import { Home } from '@pages';
import { cleanup, render } from '../renderTest';

afterEach(cleanup);

const mock = new MockAdapter(axios);

describe('<Home />', () => {
  describe('Simple Render', () => {
    it('should take a snapshot', () => {
      const { asFragment } = render(<Home />);

      expect(asFragment()).toMatchSnapshot();
    });

    it('should render without errors', () => {
      const component = render(<Home />);

      expect(component).toBeTruthy();
    });
  });
});
