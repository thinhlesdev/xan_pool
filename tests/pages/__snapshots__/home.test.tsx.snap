// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home /> Simple Render should take a snapshot 1`] = `
<DocumentFragment>
  <div
    class="page-home"
  >
    <form>
      <input
        type="text"
      />
      <button
        placeholder="username"
        type="submit"
      >
        Search
      </button>
    </form>
  </div>
</DocumentFragment>
`;
